src/app/(dashboard)/explorer/page.tsx

```diff
@@ -110,16 +106,16 @@ async function YourGuildsSection() {
 }
 
 async function YourGuilds() {
-  const auth = await getTokenFromCookie();
+  const auth = await tryGetToken();
   if (!auth) return;
 
-  const { items: myGuilds } = await getAssociatedGuilds({
-    userId: auth.userId,
+  const { items: associatedGuilds } = await getAssociatedGuilds({
```

Comment:
I'd put the `tryGetToken` call inside the `getAssociatedGuilds` function, since we only need the token for this request, wdyt?